server/routes/prisonerDetailsMiddleware.ts

```diff
@@ -61,7 +63,19 @@ export default function prisonerDetailsMiddleware({ rpService }: Services) {
       }
     }
 
+    try {
```

Comment:
I think we should add a separate middleware for this rather than wedging it in this one